src/api/treeTrackerApi.js

```diff
@@ -407,28 +431,43 @@ export default {
   /*
    * Capture Tags
    */
-  createCaptureTags(captureId, tags) {
+  createCaptureTags(capture_id, tags) {
```

Comment:
Again, I'd prefer to use camel case for function parameters.

Reply:
Noted.